public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php

```diff
@@ -258,6 +258,7 @@ public function metabox_status( $post ) {
 	 *
 	 * @return string
 	 */
+	// phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
```

Comment:
I think it's ok to say `set_locale_to_en_us` here instead of adding the `phpcs:ignore`. Not a big deal either way though.